### PR TITLE
Feature/analysis flow

### PR DESCRIPTION
**Pull request by nerik**
Implements https://github.com/NASA-IMPACT/veda-config-ghg/issues/129

### CODE REVIEW

File: app/scripts/components/analysis/results/use-analysis-params.ts [hunk 1]
```diff
@@ -25,8 +25,8 @@ type AnyAnalysisParamsKey = keyof AnalysisParams;
 type AnyAnalysisParamsType = Date | DatasetLayer[] | FeatureCollection<Polygon>;
 
 const initialState: AnalysisParamsNull = {
-  start: undefined,
-  end: undefined,
+  start: new Date(2018, 0, 1),
+  end: new Date(2022, 11, 31),
```
Comment (j08lue) [inline app/scripts/components/analysis/results/use-analysis-params.ts:29]:
```suggestion
  end: new Date(2022, 12, 31),
```

**Review by j08lue — CHANGES_REQUESTED** [review]
I see we now have a default geometry over North America, as we had discussed. I am not sure about this feature - the area is very large for a default and there could be political issues or so (you would probably need to include Hawaii, too, etc.).

So I would suggest we leave out this feature for now and leave the map blank.

File: app/scripts/components/analysis/results/use-analysis-params.ts (hunk 1 above)
Comment (nerik) [reply app/scripts/components/analysis/results/use-analysis-params.ts]: @j08lue No, in JS the date constructor takes a 0-indexed month.

Comment (j08lue) [reply app/scripts/components/analysis/results/use-analysis-params.ts:29]: oops, sorry

Comment (j08lue) [reply app/scripts/components/analysis/results/use-analysis-params.ts:29]: wait, what?!

Comment (nerik) [reply app/scripts/components/analysis/results/use-analysis-params.ts:29]: See https://stackoverflow.com/questions/2552483/why-does-the-month-argument-range-from-0-to-11-in-javascripts-date-constructor

**Pull request merged**